corehq/apps/app_manager/tests/app_factory.py

```diff
@@ -83,11 +83,12 @@ def form_updates_case(form, case_type=None, parent_case_type=None, update=None):
         else:
             case_type = case_type or form.get_module().case_type
             index = len([load for load in form.actions.load_update_cases if load.case_type == case_type])
-            action = LoadUpdateAction(
-                case_type=case_type,
-                case_tag='load_{}_{}'.format(case_type, index),
-                case_properties=update,
-            )
+            kwargs = {'case_type': case_type, 'case_tag': 'load_{}_{}'.format(case_type, index)}
+            if update:
+                kwargs['case_properties'] = update
+            if preload:
+                kwargs['preload'] = preload
+            action = LoadUpdateAction(**kwargs)
```

Comment:
My only concern is that users might use it for advanced modules and create double actions. I would prefer to just rename the function to `form_requires_case`